PR: https://github.com/odoo/odoo/pull/

From: 931afc2ff1870aff34a2f4ec5ff12374aa6cd42a
From: Haresh Shyara
Date: 2017-01-06 14:58:05

Structural Changes: 5
Total Changes: 7

[IMP] mrp: Improve the settings form view

================================= pseudo patch: =================================

--- a/addons/hr_expense/models/res_config.py
+++ b/addons/hr_expense/models/res_config.py
@@ -8,6 +8,13 @@ class HrExpenseConfigSettings(models.TransientModel):
 
     alias_prefix = fields.Char('Default Alias Name for Expenses')
     alias_domain = fields.Char('Alias Domain', default=lambda self: self.env["ir.config_parameter"].sudo().get_param("mail.catchall.domain"))
+    group_analytic_accounting = fields.Boolean(string='Analytic Accounting',
+        implied_group='analytic.group_analytic_accounting')
+    group_uom = fields.Boolean("Units of Measure",
+        implied_group='product.group_uom')
+    default_alias_email = fields.Boolean(string='Let your employees record expenses by email', default_model='hr.expense.config.settings')
+    module_project = fields.Boolean(string="Project")
+    module_sale = fields.Boolean(string="Customer Billing")
 
     @api.model
     def get_default_alias_prefix(self, fields):
